db/migrate/20241009175157_add_delete_flag_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddDeleteFlagToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :is_deleted, :boolean
```

Comment:
I would make this `null: false, default: false` just to prevent having 3 possible values.

Reply:
Actually ignore what I said because adding a column with a default rewrites the entire table and we have a ton of users.